### PR TITLE
feat: not found page

### PR DESCRIPTION
**Pull request by grabbou**
Fixes WEB-2504

Screenshot 2022-12-15 at 16 36 55
Screenshot 2022-12-15 at 16 36 49

### CODE REVIEW

File: src/pages/NotFound/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    flex: 1;
+    justify-content: center;
+  }
+`
+
+const PageWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex: 1;
```
Comment (grabbou) [inline src/pages/NotFound/index.tsx:33]: Thanks to recent work, finally all it takes to render a full page layout is a simple `flex: 1` yay!

File: src/pages/NotFound/index.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    padding-top: 65px;
+  }
+`
+
+export default function NotFound() {
+  const isDarkMode = useIsDarkMode()
+  const isMobile = useIsMobile()
+
+  const Title = isMobile ? ThemedText.LargeHeader : ThemedText.Hero
+  const Paragraph = isMobile ? ThemedText.HeadlineMedium : ThemedText.HeadlineLarge
```
Comment (grabbou) [inline src/pages/NotFound/index.tsx:45]: I added a note to Notion document that we should have this out of the box. In other words, each Text component should automatically adjust its style based on viewport and that should be aligned with our design library and Figma. Would save us a lot of time building responsive websites.

For now, I am just repeating our current approach, but eventually, we will get rid of that.

File: src/pages/App.tsx [hunk 3]
```diff
@@ -201,7 +203,11 @@ export default function App() {
           <Suspense fallback={<Loader />}>
             {isLoaded ? (
               <Routes>
+                <Route path="*" element={<Navigate to="/not-found" replace />} />
```
Comment (vm) [inline src/pages/App.tsx:206]: this should go at the bottom?

File: src/pages/NotFound/index.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+`
+
+const PageWrapper = styled(Container)`
+  flex: 1;
+  justify-content: center;
+  gap: 50px;
+
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.md}px) {
+    justify-content: space-between;
+    padding-top: 65px;
```
Comment (vm) [inline src/pages/NotFound/index.tsx:36]: 64

File: src/pages/NotFound/index.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    <PageWrapper>
+      <Trace page="404-page" shouldLogImpression>
+        <Header>
+          <Container>
+            <Title>404</Title>
+            <Paragraph color="textSecondary">
+              <Trans>Page not found!</Trans>
+            </Paragraph>
+          </Container>
+          <Image src={isDarkMode ? darkImage : lightImage} alt="404 page" />
```
Comment (vm) [inline src/pages/NotFound/index.tsx:57]: the image should say "Cow" or something, not "404 page"

**Review by vm — APPROVED** [review]

File: src/pages/NotFound/index.tsx (hunk 5 above)
Comment (grabbou) [reply src/pages/NotFound/index.tsx]: lil uni yea

**Pull request merged**